connectors/csv/src/test/java/com/datastax/oss/dsbulk/connectors/csv/CSVConnectorTest.java

```diff
@@ -1069,93 +1060,157 @@ void should_honor_nullValue_when_writing() throws Exception {
             "url",
             StringUtils.quoteJson(out),
             "nullValue",
-            null,
+            StringUtils.quoteJson(nullValue),
             "header",
-            true);
+            false);
     connector.configure(settings, false);
     connector.init();
     Flux.<Record>just(
             DefaultRecord.mapped(
                 "source",
                 resource,
                 IRRELEVANT_POSITION,
-                new Field[] {new DefaultMappedField("field1")},
-                new Object[] {null}))
+                new Field[] {new DefaultMappedField("field1"), new DefaultMappedField("field2")},
```

Comment:
We add a second field just to make sure that at a line will always be written even if field1 does not contain anything.